packages/utils/src/index.ts

```diff
@@ -2,6 +2,7 @@ export * from "./yaml/index.js";
 export * from "./assert.js";
 export * from "./base64.js";
 export * from "./bytes.js";
+export * from "./command.js";
```

Comment:
The `@lodestar/utils` package is integral to be used in `lightclient` and `prover`. So would be nice if it remains browser compatible and adding `yargs` and CLI utils may break the bundlers. 

We should try to export Node.js specific utils from a different names export and avoid from default export. May be `@lodestar/utils/cli` could be an option.

Reply:
These are just types, and a utils function which takes in yargs as an argument. I don't see a reason why we would have to export this separately

Reply:
> We should try to export Node.js specific utils from a different names export and avoid from default export. May be @lodestar/utils/cli could be an option.

I do agree that this is something we should consider, we already use this pattern in the api package.

But for the changes in this PR it is not relevant as it only adds shared types, there is no risk of breaking browser compatibility